app/src/main/res/values/strings.xml

```diff
@@ -9,7 +9,7 @@
     <string name="delete_item">Delete Item</string>
     <string name="title">Title</string>
     <string name="default_price">Price (default: 0.0)</string>
-    <string name="display_price">$ %.2f</string>
+    <string name="display_price">CHF %.2f</string>
```

Comment:
We're using CHF instead of USD now? Also should we put the currency in a string resource?